src/utils/bitsForNumber.ts

```diff
@@ -27,4 +27,13 @@ export function bitsForNumber(src: bigint | number, mode: 'int' | 'uint'): numbe
     } else {
         throw Error(`invalid mode. Got ${mode}`);
     }
-}
\ No newline at end of file
+}
+
+export function countBits64(x: number): number {
```

Comment:
"countBits64" should be considered as a separate util and should be placed into another file